### PR TITLE
Issue #588 added favicons for all devices

### PR DESCRIPTION
**Pull request by Kornil**
Included code on <head> for all devices that require a special .png format.

made a new favicon.ico file with 16x16 24x24 32x32 48x48 and 64x64 px .png included(for all desktop browsers).

On metro tile the logo has a white background.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,6 +14,28 @@
   <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
 
   <%= csrf_meta_tags %>
+  
+  <!-- favicon for every device that needs it defined -->
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="./../../assets/favicon/apple-touch-icon-57x57.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="./../../assets/favicon/apple-touch-icon-114x114.png" />
```
Comment (rickr) [inline app/views/layouts/application.html.erb]: Hello @Kornil - 

Can you convert these to use the favicon_link_tag method?

http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-favicon_link_tag

**Comment by Kornil** [comment]
Hi @rickr thanks for your suggestion, but I'm a bit confused, do you want me to format the link tags like this:

`<link href="/assets/myicon.ico" rel="shortcut icon" type="image/x-icon" />`

and simply using /assets as folder? I'm quite new with ruby and rails so forgive me if this is an obvious question, I want to be sure I'll do the right thing.

Francesco

**Comment by rickr** [comment]
This is a rails ERB file which means it can include ruby to be executed. 

The method documented [here](http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-favicon_link_tag) allows us to create favicon tags dynamically while using rails' asset pipeline. 

I think something like `favicon_link_tag 'favicon/apple-touch-icon-57x57.png', rel: 'apple-touch-icon-precomosed', type: 'image/png', sizes: '57x57'` might work.

**Comment by rickr** [comment]
Hello @Kornil, 

Are you going to be able to get to these changes?

**Comment by rickr** [comment]
Closing due to inactivity. Please feel free to reopen when you're able to make the requested changes!

**Pull request closed without merging**